Giveaways.md

```diff
@@ -0,0 +1,15 @@
+# Giveaways
+From time to time Devstaff has the opportunity to perform some giveaways during the meetups. The prizes are provided by [sponsors](https://github.com/devstaff-crete/DevStaff-Heraklion/blob/master/Sponsors.md) in return of advertising.
+
+Here we describe the process, the tools and the rules we use for these giveaways.
+
+# Prizes list
+All prizes will be written down at each meetup's announcement at [meetup.com](https://www.meetup.com/devstaff/). Moreover, at the intro presentation, when sponsors are presented, each prize will be explicitly mentioned as far as the type, the amount of prizes and the sponsor of each one.
+
+# Participants eligibility
+All participants of a meetup are eligible for the raffling process. Org team member's that atend the meetup (we prefer to provide more prizes to the community and have a chance to win them rather than benefit things as org teams members 😉), speaker(s) as well as meetup's attendees will participate in the raffling process. The participants of a meetup are the members that have joined our meetup's place (in case of physical meetups) or our communication tool (in case of remote meetups).
+
+# Raffle Process
+During the meetup (especially during the intro and at the end of each meetup) a link (to a shared Google Sheet file) will be shared where they will be able to add their name & email in order to participate at the raffling process. A number will be assigned to each participant and then we shall use the [random.org](https://www.random.org/) to generate one number per giveaway prize. Each participant's unique number that corresponds to the random number will win the prize. All prizes will be given in the order mentioned at the top of the sheet with all participants' names.  
```

Comment:
May I suggest the following rewrite to this block:
`During the meetup (especially during the intro and at the closing phase) a link (to a shared Google Sheet file) will be shared where everyone present will be able to add their name & email in order to participate to the raffling process. A number will be assigned to each participant's entry and then we shall use the [random.org](https://www.random.org/) to generate one number per giveaway prize. Each participant's unique number corresponding to the random number produced will win the prize. All prizes will be given in the order mentioned at the top of the sheet with all participants' names. `

Reply:
Just updated by https://github.com/devstaff-crete/DevStaff-Heraklion/pull/462/commits/b5477d5ab73b8780e5140cb690798434d8fcf254